src/main/java/hinlok/command/Command.java

```diff
@@ -0,0 +1,16 @@
+package hinlok.command;
+
+import hinlok.exceptions.HinlokException;
+import hinlok.file.TaskFile;
+import hinlok.tasks.TaskList;
+import hinlok.ui.Ui;
+
+public abstract class Command {
```

Comment:
Great idea to separate each of the individual commands into their own separate classes!